### ISSUE TITLE
`Collection.has_uid(self, uid)` method is not defined in the published pip3 package for Radicale 2.1.10

### ISSUE THREAD

**Issue by yoonsikp** [Kozea/Radicale#888]
Error Message from Radicale:
```
Oct 30 21:58:21 naut.ca env[19765]: [7f04888e4700] ERROR: An exception occurred during MOVE request on '/yoonsik/83AC2C70-8766-4EE7-AE69-6E195127B351/FB4D14D2-A735-46CF-B427-941C29FEA85B.ics': 'Collection
Copy' object has no attribute 'has_uid'
```

Missing function from Collection class, taken from github source:
```
    def has_uid(self, uid):
        """Check if a UID exists in the collection."""
        for item in self.get_all():
            if item.uid == uid:
                return True
        return False
```
Workaround for Debian 9:
`sudo nano /usr/local/lib/python3.5/dist-packages/radicale/storage.py`
  Paste in the above function into the Collection class.
  Restart radicale.

_Originally posted by @yoonsikp in https://github.com/Kozea/Radicale/issues/842#issuecomment-434537122

**Comment by yoonsikp** [comment]
Hello, just wanted to explain that this is a pretty serious bug since it causes data loss when moving calendar events, and that this bug exists in a release copy of Radicale version 2.1.10. This error does not occur with 2.1.9, indicating a regression. Also, I want to stress the fact that it is extremely easy to fix, and that I have tested out the potential fix already.

**Comment by Tntdruid** [comment]
I dont get that error on my system

**Comment by yoonsikp** [comment]
So I extracted Radicale 2.1.10 from the github releases. In file `__init__.py` on Line 723, there is a function definition for do_MOVE which is called whenever the macOS Calendar client wants to change an event from one calendar to another.

```
  def do_MOVE(self, environ, base_prefix, path, user):
```
In that function on Line 771, there is a method call on Collection.has_uid()
```
to_collection.has_uid(item.uid)):
```

I have searched for the method `has_uid` in the Collection class (in storage.py), and it does not exist, which explains the error message `has no attribute 'has_uid'`.

Some other details which I forgot to mention:

Client OS: macOS 10.13.6
Host OS: Debian 9
Radicale Version: 2.1.10
Description: Bug occurs with and without nginx proxy
